src/main/java/jenkins/plugins/clangscanbuild/publisher/ClangScanBuildPublisherDescriptor.java

```diff
@@ -41,10 +40,7 @@ public String getDisplayName() {
 
 	@Override
 	public boolean isApplicable( @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobType ){
-		if( !FreeStyleProject.class.isAssignableFrom( jobType ) ){
-			System.err.println( "Clang scan-build ERROR: Expected FreeStyleProject but was: " + jobType + " at Publisher Descriptor" );
-		}
-		return AbstractProject.class.isAssignableFrom( jobType );
+		return true;
```

Comment:
Careful now.... the Descriptor doesn't have the type information, so what will happen is that the descriptor will be visible in types that do not extend from `AbstractProject`... so that is a bug IMHO... It's fine to remove that println because that will basically spam the console every time you display a config page which displays build steps, but you do need to `return AbstractProject.class.isAssignableFrom( jobType )`